scripts/install/autobrr.sh

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+# autobrr installer
+# ludviglundgren 2021 for Swizzin
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+app_reqs=("curl" "sqlite3")
+apt_install "${app_reqs[@]}"
+
+users=($(_get_user_list))
+
+if [[ -n $1 ]]; then
```

Comment:
You will need to modify the useradd function inside `script/box` in order for it to trigger this. Also, please make a case for the password changing mechanism in the same file, so that the passwords are kept in sync everywhere when they're changed

Reply:
Would that be added here? https://github.com/swizzin/swizzin/blob/6f07eb7f1ed54ccea217d01d45c658b1fbc8b1be/scripts/box#L306-L331

Re the password changing mech, I'll add a function for that to `autobrrctl`.

Reply:
Yup, that's the one for `box adduser`. `box chpasswd` and `box deluser` work in a similar-ish fashion

Reply:
It only seems to be needed to add in the list at 306 right? Because the line 317 would trigger it, so extra check needed?

Reply:
Yup! If it has no special procedure, all it needs is to be in the list